source/sass/patterns/molecules/info-bar.scss

```diff
@@ -0,0 +1,61 @@
+@import "../../variables/color";
+@import "../../variables/font";
+@import "../../mixins/positioning";
+@import "../../mixins/spacing";
+@import "../../mixins/typography";
+
+$_info-bar-gutter: 8px;
+$_info-bar-image-offset: $font-size;
+$_info-bar-image-fudge-factor: 4px;
+$_info-bars: (
+  (type: attention, background-color: $color-attention),
+  (type: correction, background-color: $color-attention),
+  (type: info, background-color: $color-information),
+  (type: success, background-color: $color-success),
+  (type: warning, background-color: $color-warning),
+);
+
+.info-bar {
+  color: $color-text-reverse;
+  font-family: $font-secondary;
+  font-weight: 600;
+  @include set-font-size-and-line-height(scale(-1));
+  @include padding($baselinegrid-space-extra_small $_info-bar-gutter);
+  text-align: center;
+  position: relative;
+
+  @supports (display: flex) {
+    position: static;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  a {
+    color: inherit;
+    text-decoration: underline;
+  }
+}
+
+.info-bar__content {
+  @supports (display: flex) {
+    display: flex;
+    align-items: center;
+  }
+}
+
+.info-bar__image {
+  @include margin($_info-bar-image-offset, inline-end);
```

Comment:
<img width="388" alt="Screenshot 2019-06-25 at 15 54 25" src="https://user-images.githubusercontent.com/1784740/60109048-8f1a5400-9761-11e9-890f-9a6df2f4415a.png">

Can be solved in this case (Chrome) by changing

```css
margin-inline-end: 1rem;
```

to

```
margin-left: 0;
margin-right: 0;
margin-inline-end: 1rem;
```

Reply:
The spacing mixins could probably do with this tidy up generally 👍